docs/ciag-kpis/CIAG KPI2.mermaid

```diff
@@ -0,0 +1,140 @@
+%%
```

Comment:
Proposed diagram can be viewed [here](https://github.com/ministryofjustice/hmpps-education-and-work-plan-api/blob/feature/RR-1166-kpi2-sequence-diagram/docs/ciag-kpis/CIAG%20KPI2.mermaid)